### PR TITLE
improvement(documentation): enable mermaid diagrams rendering

### PR DESCRIPTION
**Pull request by oyo**
ref:  CPLP-2431

### CODE REVIEW

**Comment by oyo** [comment]
> IMO they all look fine with the exception of 'heap' because I can't find the source: https://www.npmjs.com/package/heap Can you find a source (release or at least a tag) for the version 0.2.7?

@evegufy there is a tagged version 0.2.6 https://github.com/qiao/heap.js/tree/0.2.6 - the only addition in 0.2.7 was adding a LICENSE file and bumping the package.json. Would that be okay to use?

**Comment by evegufy** [comment]
> > IMO they all look fine with the exception of 'heap' because I can't find the source: https://www.npmjs.com/package/heap Can you find a source (release or at least a tag) for the version 0.2.7?
> 
> @evegufy there is a tagged version 0.2.6 https://github.com/qiao/heap.js/tree/0.2.6 - the only addition in 0.2.7 was adding a LICENSE file and bumping the package.json. Would that be okay to use?

@oyo Thanks for checking! I'm not sure if the EF team would be ok with that.
I created this issue https://github.com/qiao/heap.js/issues/32

File: public/assets/images/docs/RegistrationRoleAndPermissionMatrix.png
Comment (evegufy) [inline public/assets/images/docs/RegistrationRoleAndPermissionMatrix.png:1]: same here, the update of the png has nothing to with mermaid enabling, it's just in the same commit.

File: scripts/build.sh
Comment (evegufy) [inline scripts/build.sh:1]: Thanks for adding the header!

**Pull request merged**